templates/contracts/contract-table-template.ejs

```diff
@@ -1,3 +1,4 @@
+[comment]: <> (This is an auto-generated table with CNS smart-contract addresses)
```

Comment:
This is nice. Could we also add a similar comment at the top of `src/domain-registry-essentials/cns-smart-contracts.md`, to point developers to `templates/cns-smart-contracts-template.md`?

Reply:
✅